test/unit/test_arguments.py

```diff
@@ -21,41 +21,36 @@
 from itertools import zip_longest
 
 
+# TODO: add test for new prepare_data CLI
```

Comment:
we probably should do this :)

Reply:
actually we do have a test_prepare_data_cli_args, so I'll remove the TODO.